README.md

```diff
@@ -138,7 +138,7 @@ In order to compile with tests define `BUILD_TESTS=true`
 mkdir -p arbitration_graphs/build
 cd arbitration_graphs/build
 cmake -DBUILD_TESTS=true ..
-cmake --build .
+cmake --build . -j9
```

Comment:
Why 9 btw? :D

Reply:
Kind of arbitraty, but at least more than one.

A very rough suggestion was once to use `#threads + 1`, nowadays `2 * #threads` seems to also work for many. Anyhow, it's not too crazy to assume a modern dev has >8 threads available (4 cores with hyperthreading).